### PR TITLE
Adding python-libsbml package

### PR DESCRIPTION
**Pull request by fbergmann**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by Midnighter** [comment]
@cdiener I think the bioconda package you created is causing a conflict here. I'm a little surprised since they are separate conda channels but maybe we can resolve this by closing down the bioconda one? Is that possible @conda-forge/bioconda-recipes or does the recipe need to be preserved?

**Comment by cdiener** [comment]
Unfortunately bioconda is an archive as well so they don't remove packages. We may be ae to label it broken after that one is pulled. Either way the goal here is to make an updated recipe for all platforms which is not possible in bioconda. I'm pretty sure the maintainers will understand.

File: recipes/python-libsbml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+
+  host:
+    - python
+    - setuptools
+    - pip
+    - cython
+    - swig >=3.0.12
```
Comment (scopatz) [inline recipes/python-libsbml/meta.yaml:25]: Wait - this uses swig & cython?!

File: recipes/python-libsbml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package:
+  name: python-libsbml
+  version: "5.18.3"
+
+source:
+  url: https://github.com/sbmlteam/python-libsbml/releases/download/v5.18.3/python-libsbml.tar.gz
```
Comment (scopatz) [inline recipes/python-libsbml/meta.yaml:6]: Please template this do the bot can do auto-updates: `https://github.com/sbmlteam/python-libsbml/releases/download/v{{ version }}/python-libsbml.tar.gz`

File: recipes/python-libsbml/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+package:
+  name: python-libsbml
+  version: "5.18.3"
```
Comment (scopatz) [inline recipes/python-libsbml/meta.yaml:3]: please remove quotes

**Comment by scopatz** [comment]
The bioconda piece is just to let the bioconda folks know that they also have this on their channel

File: recipes/python-libsbml/meta.yaml (hunk 1 above)
Comment (cdiener) [reply recipes/python-libsbml/meta.yaml:25]: I don't think it needs cython.

**Comment by bgruening** [comment]
> Unfortunately bioconda is an archive as well so they don't remove packages. We may be ae to label it broken after that one is pulled. Either way the goal here is to make an updated recipe for all platforms which is not possible in bioconda. I'm pretty sure the maintainers will understand.

We don't need to label it as broken as long as the conda-forge on is more up-to-date. conda-forge has priority over bioconda, even in bioconda :)

We will just remove the recipe from bioconda-recipe github, so that the bot is not updating it anymore.

**Comment by bgruening** [comment]
Merci!

**Pull request merged**